lib/Math/Matrix.pm6

```diff
@@ -181,6 +188,18 @@ method is-orthogonal(Math::Matrix:D: --> Bool) {
     self.dotProduct( self.T ) ~~ Math::Matrix.identity( $!row-count );
 }
 
+
+method is-positive-definite (Math::Matrix:D: --> Bool) { # with Sylvester's criterion
+    return False unless self.is-square or min self.size < 1;
+    return False unless self.determinant > 0;
+    my $sub = self.cone;
```

Comment:
gives me matrix object with same values. its not perfect solution but I don't know how else handle logic and references.